c7n/resources/ec2.py

```diff
@@ -898,6 +898,64 @@ def process_resource_set(self, client, resources):
             r[self.annotation] = info_map.get(r['InstanceId'], {})
 
 
+@actions.register('set-monitoring')
+class MonitorInstances(BaseAction, StateTransitionFilter):
+    """Action on EC2 Instances to enable/disable detailed monitoring
+
+    The differents states of detailed monitoring status are :
+    'disabled'|'disabling'|'enabled'|'pending'
+    (https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_instances)
+
+    :Example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: ec2-detailed-monitoring-activation
+            resource: ec2
+            filters:
+              - Monitoring.State: disabled
+            actions:
+              - type: set-monitoring
+                state: enable
+    References
```

Comment:
this needs to have a new line separating it from the last one, I think. the doc builds are failing due to restructured text error/warnings from sphinx atm.

Reply:
yeah this looks like the issue per the traceback

```
__________________________ test_doc_examples[aws-AWS] __________________________
[gw0] linux2 -- Python 2.7.17 /home/vsts/work/1/s/.tox/py27/bin/python
Traceback (most recent call last):
  File "/home/vsts/work/1/s/tests/test_doc_examples.py", line 95, in test_doc_examples
    policies, duplicate_names = get_doc_policies(provider.resources)
  File "/home/vsts/work/1/s/tests/test_doc_examples.py", line 60, in get_doc_policies
    data = yaml_load(ptext)
  File "/home/vsts/work/1/s/c7n/utils.py", line 107, in yaml_load
    return yaml.load(value, Loader=SafeLoader)
  File "/home/vsts/work/1/s/.tox/py27/lib/python2.7/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/home/vsts/work/1/s/.tox/py27/lib/python2.7/site-packages/yaml/constructor.py", line 66, in get_single_data
    node = self.get_single_node()
  File "/home/vsts/work/1/s/.tox/py27/lib/python2.7/site-packages/yaml/composer.py", line 39, in get_single_node
    if not self.check_event(StreamEndEvent):
  File "/home/vsts/work/1/s/.tox/py27/lib/python2.7/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/vsts/work/1/s/.tox/py27/lib/python2.7/site-packages/yaml/parser.py", line 174, in parse_document_start
    self.peek_token().start_mark)
ParserError: expected '<document start>', but found '<scalar>'
  in "<unicode string>", line 9, column 5:
        References
        ^
```